packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -728,6 +728,17 @@ export function TransferPanel() {
   )
   const { status: gasEstimationStatus } = gasSummary
 
+  const requiredGasFees = useMemo(
+    // For SC wallets, the relayer pays the gas fees so we don't need to check in that case
+    () =>
+      isSmartContractWallet
+        ? isDepositMode
+          ? gasSummary.estimatedL2GasFees
+          : 0
+        : gasSummary.estimatedTotalGasFees,
+    [isSmartContractWallet, gasSummary.estimatedTotalGasFees]
+  )
```

Comment:
```suggestion
  const requiredGasFees = useMemo(
    // For SC wallets, the relayer pays the gas fees so we don't need to check in that case
    () => {
      if (isSmartContractWallet) {
        if (isDepositMode) {
          return gasSummary.estimatedL2GasFees
        }
        return 0
      }
      return gasSummary.estimatedTotalGasFees
    },
    [isSmartContractWallet, isDepositMode, gasSummary]
  )
```

Reply:
tested deposit & withdrawal with 0 ETH safe on goerli and arb goerli respectively
works with the suggested change

Reply:
```suggestion
  const requiredGasFees = useMemo(
    // For SC wallets, the relayer pays the gas fees so we don't need to check in that case
    () => {
      if (isSmartContractWallet) {
        if (isDepositMode) {
          // L2 fee is paid in callvalue and still need to come from the wallet for retryable cost estimation to succeed
          return gasSummary.estimatedL2GasFees
        }
        return 0
      }
      return gasSummary.estimatedTotalGasFees
    },
    [isSmartContractWallet, isDepositMode, gasSummary]
  )
```